roles/postgresql/templates/pg_hba.conf.j2

```diff
@@ -0,0 +1,5 @@
+{{ ansible_managed | comment }}
+
```

Comment:
Can we expand this template to include all the changes we're making to `pg_hba.conf` elsewhere?

Reply:
we have one set for the postgresql servers and another for the client connections. Separating these two keeps me sane.